autoscale-core/src/main/java/com/hpe/caf/autoscale/core/GovernorImpl.java

```diff
@@ -43,16 +52,56 @@ public GovernorImpl(final int stageOneLimit, final int stageTwoLimit, final int
         this.stageOneShutdownPriorityLimit = stageOneLimit;
         this.stageTwoShutdownPriorityLimit = stageTwoLimit;
         this.stageThreeShutdownPriorityLimit = stageThreeLimit;
+        this.scalerThreads = new HashMap<>();
+    }
+
+    @Override
+    public boolean makeRoom(final String serviceRef) throws ScalerException
+    {
+        final AdvancedInstanceInfo lastInstanceInfo = instanceInfoMap.getOrDefault(serviceRef, null);
+        if (lastInstanceInfo == null) {
+            return false;
+        }
+        final double percentageDifference = lastInstanceInfo.getPercentageDifference();
+        final Map<String, AdvancedInstanceInfo> possibleVictims = instanceInfoMap.entrySet().stream()
+            .filter(e -> e.getValue().getPercentageDifference() > -1)
```

Comment:
What is so special about `-1`, given that this is a `double`?  Is this meant to be `>= 0`?